frontend/src/app_context.tsx

```diff
@@ -7,8 +7,20 @@ import {
 } from "react";
 import { LoginState } from "./constants";
 
+export enum WordVaultFontStyle {
```

Comment:
Since there's an external module that now depends on these value for app logic, I wanted to make sure the correct values were defined exhaustively

I'm also open to making this something like

```ts
type FontFamily = 'sans-serif' | 'monospace';
type WordVaultFontStyle = FontFamily | 'tiles'
```

Since right now `WordVaultFontStyle` is a slightly odd abstraction where 2 of the values are `font-family` literals that can be passed directly as css and the other is not

Reply:
My preference would be to do something similar to what I did on WordWalls which is have tile style just be different colors, and then the tiles can either have serif or non-serif fonts on them. But there I also had a bool setting that is something like `useTiles`. 

Now more than 10 years later I am open to having a better API though so input welcome.

Reply:
Looping back now -- I like `tileStyle` being non-null to enable tile display (eventually supporting potentially different colors) with the font being a distinct option. Will implement that!